setup.py

```diff
@@ -301,11 +297,9 @@ def run(self):
                     'pycbc.results': find_files('pycbc/results'),
                     'pycbc.tmpltbank': find_files('pycbc/tmpltbank')},
     ext_modules = ext,
-    python_requires='>=2.7',
+    python_requires='>=3.7',
```

Comment:
Just to check, what happens if someone tries to install this using python2, or 3.6. Will pypi know to take an earlier version? Will it fail if installing from source?

Reply:
pip will install an earlier version and will not allow installing any future released versions. Attempting a source installation will also produce an error message on earlier versions of python.